server/tests/tasks/test_bitbucket_change_gating_handler.py

```
@@ -0,0 +1,264 @@
+        _run()
+        delay.assert_not_called()
+
+    def test_missing_fields_skip(self, gating_env):
+        _, _, delay = gating_env
+        payload = _payload()
+        del payload["pullrequest"]["source"]
+        _run(payload=payload)
+        delay.assert_not_called()
+
+    def test_duplicate_delivery_skips_without_enqueue(self, gating_env):
+        _, redis, delay = gating_env
```

Ruff (0.16.1)

[warning] Unpacked variable `redis` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)